cypress/integration/ColorPicker.spec.ts

```diff
@@ -194,32 +194,38 @@ describe('ColorPicker', () => {
       });
 
       context('when a swatch is clicked', () => {
-        const color = '8660d1';
+        const color = '#ff5347';
+
         beforeEach(() => {
-          cy.get(`.wdc-color-picker--color-${color}`).click();
+          getSwatch(color).click();
         });
 
         it('should update the color input to the swatch color', () => {
-          getColorInput().should('have.value', color.toUpperCase());
+          getColorInput().should('have.value', color.slice(1).toUpperCase());
         });
 
         it('should have check icon', () => {
-          getColorInput().click();
-          cy.get(`.wdc-color-picker--color-${color}`)
+          getColorInput().focus();
```

Comment:
Not sure if this matters too much, it was just odd that we used `focus()` to open the picker and then `click()` to re-open it.